swerve100/src/main/java/frc/robot/autonomous/DriveToWaypoint2.java

```diff
@@ -30,7 +31,7 @@
  */
 public class DriveToWaypoint2 extends CommandBase {
     private static final TrapezoidProfile.Constraints rotationConstraints = new TrapezoidProfile.Constraints(6, 12);
-
+    private AHRSClass m_gyro;
```

Comment:
this field is never assigned.  you should assign to it in the constructor.  also make it final.